### ISSUE TITLE
GNS3 GUI doesn't launch

### ISSUE THREAD

**Issue by starcraft66** [NixOS/nixpkgs#81048]
**Describe the bug**
When launching the gns3 gui by running `gns3`, there is a stacktrace.

**To Reproduce**
1. Install the `gns3-gui` package
2. Run `gns3`

**Expected behavior**
GNS3 Opens

**What actually happens**

```
tristan@luna:~$ gns3
Traceback (most recent call last):
  File "/nix/store/zl1l3nrhc4lf7ibzn1ws97nwi0ikglmd-python3.7-gns3-gui-2.1.21/bin/.gns3-wrapped", line 7, in <module>
    from gns3.main import main
  File "/nix/store/zl1l3nrhc4lf7ibzn1ws97nwi0ikglmd-python3.7-gns3-gui-2.1.21/lib/python3.7/site-packages/gns3/main.py", line 40, in <module>
    from gns3.utils.get_resource import get_resource
  File "/nix/store/zl1l3nrhc4lf7ibzn1ws97nwi0ikglmd-python3.7-gns3-gui-2.1.21/lib/python3.7/site-packages/gns3/utils/get_resource.py", line 21, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
```

**Comment by primeos** [comment]
@starcraft66 thanks for your report, this should be fixed once d0b3ab3b249fb62ee1997b34bbfd65600231769f is available in the stable-channel (tested it in a VM). 

In case you now get a segmentation fault instead you also have to install `qt5Full`.

**Comment by starcraft66** [comment]
Thanks, it appears the fix was very simple. I am new to Nix and NixOS so I mistakenly looked at the nixpkgs master branch where setuptools was already added to the build which confused me.
